AOSC-Dev/aosc-os-abbs
# feeluown: 3.7.3+git20210211

Topic Description
-----------------
- Update `FeelUOwn` to 3.7.3+git20210211.
- Update `qasync` to 0.13.0.
- Update `feeluown-netease` to 0.5.
- Add `pydantic`
- Drop `feeluown-download`

Package(s) Affected
-------------------
- Add: `pydantic`
- Update: `qasync`
- Update: `feeluown`
- Update: `feeluown-netease`
- Drop :`feeluown-download`

Security Update?
----------------
No 

Build Order
-----------
- `pydantic`, `qasync` => `feeluown` => `feeluown-netease`

Architectural Progress
----------------------
 - [x] Architecture-independent `noarch`

----

After the pull request is merged, all package(s) affected must be rebuilt against the `stable` Git tree and environment (only `stable` repository should be enabled in `sources.list`). This section marks the progress above.

Please, make sure the list of architectures below matches the ones above.

Post-Merge Architectural Progress
---------------------------------
 - [ ] Architecture-independent `noarch`
